Technical-Steering-Committee/contributing.md

```diff
@@ -0,0 +1,94 @@
+# Contributing and roles
+
+This document explains roles within Zowe and the ways and requirements for contribution. 
+
+The goal of the document is to outline key roles within Zowe with the guidance for the squads around the privileges associated with the roles. Each squad can update the privileges as needed. The exact structure and rules if changed will be in the CONTRIBUTING.md in the repositories belonging to the squad. 
+
+The roles will be reflected in Teams on the Zowe GitHub ([Github Teams](https://github.com/orgs/zowe/teams)). Each Squad will have a Committers, Contributors and Administrators teams.  
+
+## Roles
+
+- Guest
+- Contributor
+- Committer
+- Administrator
+
+### Guest
+
+Anyone, no privileges required. 
+
+- Can read code
+- Can file issues
+- Can create PRs
+- Can comment on issues
+
+### Contributor
+
+Accepted as a Contributor upon request in the squads channel. Automatically approved after filing an issue and/or creating PR. 
+
+Everything Guest can do and on top of that:
+
+- Can review PRs
+- Edit issues and PRs (labels, closing)
+- Can submit blog posts for approval
+- Can submit videos for approval
+
+### Committer
+
+Contributors can become Committers via being voted into the role by Committers
+
+Everything Contributor can do and on top of that:
+
+- Can make branches
+- Can create releases
+- Can merge code
+- Can delete unprotected branches
+- Can approve blog posts
+- Can approve videos
+- Can update the website (the action of taking the latest branch and making it public, is this committer or admin?)
+- Can edit wikis
+
+### Administrator
+
+Committers can become Administrators of a squad by being voted into the role by Committers
+
+Everything Committer can do and on top of that:
+
+- Every action not already listed, such as but not limited to
+- Adding and removing users to the roles
+- Changing branch permissions
+- Changing github hooks
+- Changing repo status
+
+### Zowe administrators
+
+Selected by TSC. Needs to be voted to the position. The zowe wide administrator will have the Administrator privileges for all Zowe repositories. 
```

Comment:
Not sure what's the common practice here, like how many owners are suggested for a github org. To add people to the org, afaik it needs owner permission. If we automatically promote TSC members as Zowe administrators, then each squad will have a representative as zowe github owner, and will be able to add members by themselves.

Just the squad representative should be careful not to touch other squads permissions. I believe TSC members will understand well and behave as expected.

Reply:
Where do we keep the information?
Who are the people?
How do we vote and for how long?
Review the current owners for Zowe?